example/listenvironments/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2015 The go-github AUTHORS. All rights reserved.
```

Comment:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

The copyright date is set to the year that the file first appears in our repo, so please change this to 2022.  Thanks.

Reply:
My bad - committed the suggestion.